ui/src/app/components/cromwell-configuration-panel.tsx

```diff
@@ -62,12 +63,25 @@ const PanelMain = fp.flow(
       appsApi().listAppsInWorkspace(workspace.namespace).then(setUserApps);
     }, []);
 
+    const onDismiss = () => {
+      onClose();
+      setTimeout(() => setSidebarActiveIconStore.next('apps'), 3000);
+    };
+
     const onCreate = () => {
       if (!creating) {
         setCreating(true);
-        appsApi().createApp(workspace.namespace, defaultCromwellConfig);
-        onClose();
-        setTimeout(() => setSidebarActiveIconStore.next('apps'), 3000);
+        appsApi()
+          .createApp(workspace.namespace, defaultCromwellConfig)
+          .then(() => onDismiss())
+          .catch(() =>
+            notificationStore.set({
```

Comment:
Used `notificationStore.set({...})` instead of `withErrorModal`, because withErrorModal does not currently return a promise. I am willing to update `withErrorModal` to return a promise, if that is the preferred approach.

Reply:
(resolving because outdated)